packages/cvs-common/src/classes/testTypes/Constants.ts

```diff
@@ -209,3 +209,99 @@ export const CENTRAL_DOCS_TEST = {
 		'195',
 	],
 };
+
+export const ABANDONED_TEST = {
```

Comment:
Is this the right name? Maybe it is, but can other tests be abandoned but not make a VTG12 etc?

Reply:
It's just naming convention as I wanted to amalgamate both the VTP12 and VTG12 test type ids.

This list will be used to determine whether or not the test type id on the test result is valid to produce the certificate, assuming the flag is enabled.

Reply:
Yeah I think so, if I looked at this in a year I'd like to know that it's for a certain document, not just those are tests that can be abandoned, which is how it reads.

Reply:
I've gone for `VTG_VPT_12` in a few places, doesn't have to match but just a suggestion.

Reply:
No problem, updated.